model_analyzer/entrypoint.py

```diff
@@ -296,24 +292,49 @@ def main():
     Main entrypoint of model_analyzer
     """
 
+    global exiting
+
+    # Model Analyzer Status
+    exiting = False
+
     monitoring_metrics = [
         PerfThroughput, GPUUtilization, GPUUsedMemory, GPUFreeMemory
     ]
 
+    def interrupt_handler(signal, frame):
```

Comment:
What makes this different than 
```
try:
    # main code
except KeyboardInterrupt:
    # Exiting gracefully
```

Reply:
I didn't want the `interrupt` to interrupt the execution of the program. For example, when the Triton container is being created it will not be interrupted by the Ctrl-c. In this way, it will be still interrupted.